composeApp/src/commonMain/kotlin/networking/Fetch20Result.kt

```diff
@@ -0,0 +1,35 @@
+package networking
+
+/**
+ * {
+ * gameId: 1100, - ID Igre
+ * drawId: 838524, - ID izvlačenja
+ * za Grčki kino
+ * drawTime: 1602580200000, - vreme izvlačenja
+ * status: "active",
+ * drawBreak: 0,
+ * visualDraw: 838524,
+ * pricePoints: {},
+ * prizeCategories: [],
+ * wagerStatistics: {},
+ * }
+ */
+
+data class Fetch20Result(
+    val gameId: Int,
+    val drawId: Int,
+    val drawTime: Long,
+    private val stringStatus: String,
+    val status: DrawStatus = when(stringStatus) {
+        "active" -> DrawStatus.ACTIVE
+        "future" -> DrawStatus.FUTURE
+        else -> throw IllegalArgumentException("Status ${stringStatus} not covered by DrawStatus! Please add it")
+    },
```

Comment:
maybe this shouldn't be here, but in the actual class implementation part; it shouldn't be passable via constructor args but only computed.

move this. otherwise the whole `when` makes no sense if this is passed in.

Reply:
resolving